src/planner/Planner.tsx

```diff
@@ -236,6 +236,7 @@ export const Planner = (props: PlannerProps) => {
 
     const focusInfo = useFocusInfo();
     const focusModeEnabled = !!focusInfo;
+    const [panZoom, setPanZoom] = useState({ x: 0, y: 0, zoom: 1 });
```

Comment:
Do we need it here when we already have it in the planner context? Every component inside that react context can get it from the context

https://github.com/kapetacom/ui-web-plan-editor/blob/master/src/planner/PlannerContext.tsx#L92-L96

Reply:
No in that case we don't. I just couldn't grok it apparently. Thanks, will reuse that one instead.